homeassistant/components/netatmo/sensor.py

```diff
@@ -137,9 +150,16 @@ async def find_entities(data_class_name):
             _LOGGER.debug(
                 "Adding module %s %s", module.get("module_name"), module.get("_id"),
             )
-            for condition in data_class.get_monitored_conditions(
-                module_id=module["_id"]
-            ):
+            conditions = data_class.get_monitored_conditions(module_id=module["_id"])
+            for condition in conditions:
+                if f"{condition.lower()}_value" in SENSOR_TYPES:
```

Comment:
Since you only care about lowercase, let's just lowercase condition once at the beginning